src/main/java/racing_car/controller/GameController.java

```diff
@@ -0,0 +1,45 @@
+package racing_car.controller;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import racing_car.domain.car.Car;
+import racing_car.domain.car.Cars;
+import racing_car.view.InputView;
+import racing_car.view.OutputView;
+
+public class GameController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private Cars cars;
+
+
+    public GameController(Scanner scanner) {
+        inputView = new InputView(scanner);
+        outputView = new OutputView();
+    }
+
+    public void play() {
+        String[] carNames = inputView.inputCarNames();
+        cars = new Cars(Arrays.stream(carNames)
+            .map(Car::new)
+            .collect(Collectors.toList())
+        );
+        int tryCount = inputView.inputTryCount();
+        startGame(tryCount);
+        announceWinners();
+    }
+
+    public void startGame(int tryCount) {
+        outputView.printResultMessage();
+        for (int t = 1; t <= tryCount; t++) {
```

Comment:
```suggestion
        for (int i = 0; i < tryCount; i++) {
```

for문 조건식에 들어가는 변수는 i,j,k를 쓰는게 일반적입니다.
특수한 케이스가 아니면 초기값은 0, 조건문은 < 을 사용합니다.